app/src/main/java/at/bitfire/icsdroid/ui/AddCalendarEnterUrlFragment.kt

```diff
@@ -58,6 +61,23 @@ class AddCalendarEnterUrlFragment: Fragment() {
         binding.lifecycleOwner = this
         binding.model = subscriptionSettingsModel
 
+        binding.credentialsComposable.apply {
+            // Dispose the Composition when viewLifecycleOwner is destroyed
+            setViewCompositionStrategy(
+                ViewCompositionStrategy.DisposeOnLifecycleDestroyed(viewLifecycleOwner)
+            )
+            setContent {
+                LoginCredentialsComposable(
```

Comment:
I recall the same as in https://github.com/ArnyminerZ/icsx5/pull/33#pullrequestreview-1787056878.

Reply:
yup, will do this when rebasing with dev, once the #33 has been merged :)